lib/config-initializer.js

```diff
@@ -44,6 +44,9 @@ function processAnswers(answers) {
     config.rules.quotes = [2, answers.quotes];
     config.rules["linebreak-style"] = [2, answers.linebreak];
     config.rules.semi = [2, answers.semi ? "always" : "never"];
+    if (answers.extendDefault) {
```

Comment:
Not sure about that. I think asking a bunch of style questions and then forcing other styles on the user might be a little unexpected. I think defaulting to true is probably the most we want to do here.